Dockerfile

```diff
@@ -94,7 +94,7 @@ RUN make VERSION=1.0.5
 
 ###############################################################################
 
-FROM ubuntu:noble-20240801 AS bundler
+FROM --platform=$TARGETPLATFORM scratch
```

Comment:
Esse `--platform=$TARGETPLATFORM` o Docker indicou como um pleonasmo.

Reply:
Just trying to be explicit. We enforece linux/riscv64 via CI.

Reply:
But isn't this always the case? Isn't the last image, if no `--platform` option provided, build for the target platform?

Reply:
Or we should force it via hardcoded `--paltform=linux/ricvs64` so that a local `docker build` works the same as CI.

Reply:
I think Docker also advises not to do that.

Reply:
I would simply remove this `--from` and have the target platform be specified on the CI.